quickwit/quickwit-ingest/src/ingest_v2/doc_mapper.rs

```diff
@@ -122,7 +148,7 @@ pub(super) async fn validate_doc_batch(
     doc_mapper: Arc<dyn DocMapper>,
 ) -> IngestV2Result<(DocBatchV2, Vec<ParseFailure>)> {
     if is_document_validation_enabled() {
```

Comment:
out of scope for this PR: it might be worth thinking about disabling validation when VRL is present, or to do VRL validation early. I can imagine a document-doc_mapper-VRL combo where not having VRL here causes a document to be rejected, even though it would have been accepted during actual indexation

Reply:
excellent point! I'll create an issue.